examples/example_FastMRI_Unet.py

```diff
@@ -0,0 +1,206 @@
+# %%
+"""
+Simple UNet model.
+
+This model is a simplified version of the U-Net architecture, 
```

Comment:
Please cite U net here. See how to cite here:

https://github.com/mind-inria/mri-nufft/blob/092c795c9fc251966ce0422ee40eb4419ff4faf1/examples/GPU/example_learn_samples.py#L20C108-L21C4